app/javascript/react/components/AnimalIndex.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+const AnimalIndex = (props) => {
+  return (
+    <h1>Animals</h1>
```

Comment:
No animals being displayed yet? For this feature of an animals index page, we wouldn't consider it complete if we aren't able to see that data that you set up on the backend!